front/src/applications/operationalStudies/components/MacroEditor/NGE.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react';
+
+import ngeMain from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/main.js?url';
+import ngePolyfills from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/polyfills.js?url';
+import ngeRuntime from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/runtime.js?url';
+import ngeStyles from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/styles.css?url';
+import ngeVendor from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/vendor.js?url';
+
+import 'styles/scss/applications/operationalStudies/_nge.scss';
+
+const frameSrc = `
+<!DOCTYPE html>
+<html class="sbb-lean">
+  <head>
+    <link rel="stylesheet" href="${ngeStyles}"></link>
+    <script type="module" src="${ngeRuntime}"></script>
+    <script type="module" src="${ngePolyfills}"></script>
+    <script type="module" src="${ngeVendor}"></script>
+    <script type="module" src="${ngeMain}"></script>
+  </head>
+  <body></body>
+</html>
+`;
+
+function NGE() {
```

Comment:
Use const over function 
```suggestion
const NGE = () => {
```

Reply:
(Ditto for `handleFrameLoad`)

Reply:
Done